nix-support-production/lainchan.nix

```diff
@@ -4,7 +4,36 @@ let
   app = "lainchan";
   domain = "leftypol.org";
   dataDir = "/srv/http/${app}.leftypol.org";
-  #acmeRoot = "/var/lib/acme/acme-challenge";
+  oldpkgs = import ./nixpkgs {};
+
+  leftypol_common_location_block = {
+    "~ \.php$" = {
+      root = dataDir;
+      extraConfig = ''
+            # fastcgi_split_path_info ^(.+\.php)(/.+)$;
+            fastcgi_pass unix:${config.services.phpfpm.pools.${app}.socket};
+      '';
+    };
+
+    "~* \.(?:html|json)$" = {
+      root = dataDir;
+      extraConfig = ''
+        expires 1s;
+      '';
+    };
+
+    "~* \.(jpg|jpeg|png|gif|ico|css|js|mp4|mp3|webm|pdf|bmp|zip|epub)$" = {
+      root = dataDir;
```

Comment:
Same as the other comment. Is this still necessary? If not, let's set the expiration much higher.